Formula/jpegoptim.rb

```diff
@@ -1,8 +1,8 @@
 class Jpegoptim < Formula
   desc "Utility to optimize JPEG files"
   homepage "https://github.com/tjko/jpegoptim"
-  url "https://github.com/tjko/jpegoptim/archive/RELEASE.1.4.6.tar.gz"
-  sha256 "c44dcfac0a113c3bec13d0fc60faf57a0f9a31f88473ccad33ecdf210b4c0c52"
+  url "https://github.com/tjko/jpegoptim/archive/RELEASE.1.4.7.tar.gz"
+  sha256 "9d2a13b7c531d122f360209422645206931c74ada76497c4aeb953610f0d70c1"
   license "GPL-2.0"
```

Comment:
License situation after this release is a little unclear. `COPYRIGHT` indicates GPL 2.0 or later, but `LICENSE` was updated in this release to reference GPL 3.0.

Reply:
Filed a PR to address this.